.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ e2eTest: &e2eTest
 
     - run: ./e2e-harness setup --remote=false
 
-    - run: ./e2e-harness test --test-dir=integration/test/basic
+    - run: ./e2e-harness test --test-dir=${TEST_DIR}
```

Comment:
Just handling this similar to other repos now, so the different integration tests can run in parallel.